From 379b0685c5fd130f666d0b6c0403ff1c77722736 Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 12 Dec 2021 00:45:11 +0000
Subject: [PATCH 1/6] gnu: Add tagparser

* gnu/packages/golang.scm: (go-github-com-vmihailenco-tagparser): New variable.
---
 gnu/packages/golang.scm | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 70731e2720..e484abd8e6 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -8924,3 +8924,24 @@ (define-public go-github-com-arceliar-ironwood
 @url{https://github.com/yggdrasil-network/yggdrasil-go,Yggdrasil}, but it may
 be useful for other network applications.")
     (license license:mpl2.0)))
+
+(define-public go-github-com-vmihailenco-tagparser
+  (package
+    (name "go-github-com-vmihailenco-tagparser")
+    (version "2.0.0")
+    (source
+      (origin
+        (method git-fetch)
+        (uri (git-reference
+               (url "https://github.com/vmihailenco/tagparser")
+               (commit (string-append "v" version))))
+        (file-name (git-file-name name version))
+        (sha256
+          (base32 "13arliaz3b4bja9jj7cr5ax4zvxaxm484fwrn0q6d6jjm1l35m1k"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/vmihailenco/tagparser"))
+    (home-page "https://github.com/vmihailenco/tagparser")
+    (synopsis "Tag parser for Golang")
+    (description "Simple Golang implementation of tag parser")
+    (license license:bsd-2)))
-- 
2.34.0

